react/typings/global.d.ts

```diff
@@ -78,7 +78,11 @@ declare global {
     id: string
   }
 
-  type RenderScrollOptions = ScrollToOptions | false
+  interface RenderScrollToOptions extends ScrollToOptions {
```

Comment:
`ScrollToOptions` is named like that because it directly reffers [the one in the Web API](https://developer.mozilla.org/en-US/docs/Web/API/ScrollToOptions). `RenderScrollOptions` is named like that because we think `RenderScrollToOptions` is ugly.

I don't think having both `RenderScrollToOptions` and `RenderScrollOptions` is a great idea because they both have the exact same meaning. I would use something like `ExtendedScrollToOptions` or `PagesScrollToOptions` and leave the rest as it is.

Reply:
Agree